### PR TITLE
Adds an admonition about Teleport not currently supporting Azure AD

### PR DESCRIPTION
**Pull request by ibeckermayer**

### CODE REVIEW

**Review by ptgott — COMMENTED** [review]
Does this Admonition also apply to `docs/pages/desktop-access/active-directory-manual.mdx`? 

Also, can organizations that use Azure AD take advantage of the approach documented in `docs/pages/desktop-access/getting-started.mdx` with Teleport Enterprise? If so, maybe we can add a link to the Admonition recommending this approach?

**Comment by ibeckermayer** [comment]
> Does this Admonition also apply to `docs/pages/desktop-access/active-directory-manual.mdx`?

Yes

> Also, can organizations that use Azure AD take advantage of the approach documented in `docs/pages/desktop-access/getting-started.mdx` with Teleport Enterprise? If so, maybe we can add a link to the Admonition recommending this approach?

I'm pretty sure this is the case, added in cc18a661ff8644d0b00999f3c6a92a8c38ba48a2. @probakowski @zmb3 can either of you confirm this?

File: docs/pages/desktop-access/active-directory-manual.mdx [hunk 1]
```diff
@@ -31,6 +31,20 @@ This guide requires you to have:
 - An Active Directory domain, configured for LDAPS (Teleport requires an
   encrypted LDAP connection). Typically this means installing
   [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+
+<Admonition title="Azure AD" type="warning">
+
+Microsoft's Azure Active Directory (Azure AD) offering does not support the
+Kerberos authentication protocol, which is required for Teleport's
+certificate-based authentication integration.
```
Comment (zmb3) [inline docs/pages/desktop-access/active-directory-manual.mdx:39]:
```suggestion
certificate-based authentication.
```

File: docs/pages/desktop-access/active-directory.mdx [hunk 2]
```diff
@@ -19,6 +19,19 @@ The features documented on this page are available in Teleport `10.2.6` and high
 This guide will help you configure Teleport and Active Directory in order to
 provide secure, passwordless access to Windows desktops.
 
+<Admonition title="Azure AD" type="warning">
+
+Microsoft's Azure Active Directory (Azure AD) offering does not support the
+Kerberos authentication protocol, which is required for Teleport's
+certificate-based authentication integration.
```
Comment (zmb3) [inline docs/pages/desktop-access/active-directory.mdx:26]:
```suggestion
certificate-based authentication.
```

**Review by zmb3 — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Pull request merged**